src/components/AvatarBlock/AvatarBlock.tsx

```diff
@@ -0,0 +1,50 @@
+import clsx from 'clsx';
+import React, { ReactNode } from 'react';
+import styles from './AvatarBlock.module.css';
+
+import { Avatar } from '../..';
+
+export interface Props {
+  /**
+   * Child node(s) that can be nested inside component
+   */
+  children?: ReactNode;
+  /**
+   * CSS class names that can be appended to the component.
+   */
+  className?: string;
+  /**
+   * URL or a URL fragment string that the hyperlink points to
+   */
+  href?: string;
+}
+
+/**
+ * Primary UI component for user interaction
+ */
+export const AvatarBlock: React.FC<Props> = ({
```

Comment:
I believe we're moving away from using `React.FC` https://github.com/chanzuckerberg/edu-design-system/pull/919